frontend/src/pages/ontologies/ontologiesSlice.ts

```diff
@@ -252,20 +252,14 @@ export const getOntologies = createAsyncThunk(
 export const getAllOntologies = createAsyncThunk(
     "ontologies_all",
     async (_, { rejectWithValue }) => {
-        const path = `api/v2/ontologies?size=1`;
+        const allOntologiesPath = `api/v2/cache/ontologies`;
```

Comment:
Is there any reason we can't just make the cached endpoint the only endpoint instead of having a separate API route?